extensions/builtbybit/src/get-resources.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  Detail,
+  getPreferenceValues,
+  PreferenceValues,
+  List,
+  Color,
+  Icon,
+  LaunchProps,
+  ActionPanel,
+  Action,
+} from "@raycast/api";
+import { showFailureToast, useFetch } from "@raycast/utils";
+import { API_BASE_URL } from "./utils/constants";
+
+const preferences = getPreferenceValues<PreferenceValues>();
```

Comment:
There is no such thing as `PreferenceValues` in the API, you should use the types autogenerated in `raycast-env.d.ts`